tests/dogfood.rs

```diff
@@ -3,179 +3,26 @@
 //!
 //! See [Eating your own dog food](https://en.wikipedia.org/wiki/Eating_your_own_dog_food) for context
 
-// Dogfood cannot run on Windows
-#![cfg(not(windows))]
```

Comment:
The windows and mac runners are our slowest runners in CI and also the current bottleneck. This added another 1-2min to the windows runner. So I wonder if we should only run those tests on `unix`. An  argument against that is, that contributors using windows/mac, won't see dogfood errors until they submit a PR.

Reply:
Thanks for reviewing this.

Maybe I can find a way to get the best of both worlds using a flag or environment variable.